File: 프로그래머스/lv1/신은혜/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        if(!(s.length()==4||s.length()==6)) return false;
        for(int i=0; i<s.length(); i++){
            char c = s.charAt(i); 
            if(!('0'<=c&&c<='9')) return false;
        }
        return true;
    }
}

File: 프로그래머스/lv1/leehandsub/문자열 다루기 기본.cpp
#include <string>
#include <vector>
#include <stdio.h>
using namespace std;

bool solution(string s) {
    bool answer = true;  
    for(int i=0;i<s.size();i++)
    {
        if(s[i]-'0'<0||s[i]-'0'>9)
        {
            //printf("%d ",s[i]-'0');
            return false;
        }
    }
    if(s.size()==4||s.size()==6)
        return answer;
    else
        return false;
}
